distutils/tests/test_install.py

```diff
@@ -209,16 +211,19 @@ def test_record(self):
         found = [os.path.basename(line) for line in content.splitlines()]
         expected = ['hello.py', 'hello.%s.pyc' % sys.implementation.cache_tag,
                     'sayhi',
-                    'UNKNOWN-0.0.0-py%s.%s.egg-info' % sys.version_info[:2]]
+                    'testdist-0.1-py%s.%s.egg-info' % sys.version_info[:2]]
         self.assertEqual(found, expected)
 
     def test_record_extensions(self):
         cmd = test_support.missing_compiler_executable()
         if cmd is not None:
             self.skipTest('The %r command is not found' % cmd)
         install_dir = self.mkdtemp()
-        project_dir, dist = self.create_dist(ext_modules=[
-            Extension('xx', ['xxmodule.c'])])
+        project_dir, dist = self.create_dist(
+            name="testdist",
+            version="0.1",
```

Comment:
In other places, I've argued against requiring to supply name and version, that "UNKNOWN" and 0.0.0 are good defaults for a lot of cases, tests especially. Here's an example where the test is having to introduce extra lines and values irrelevant to the behavior under test, distracting from the essential behavior.